### PR TITLE
Add a blog post announcing API changes.

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: _posts/2015-06-10-Upcoming-API-Changes.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+type: post
+published: true
+author: Ketan Padegaonkar
+---
+
+With the upcoming release of Go 15.2, we'd like to begin unifying and improving some of the existing APIs that Go supports.
+
+Go's APIs are fairly old, have an [inconsistent and unpredictable content types](https://github.com/gocd/gocd/issues/572) (csv, xml, json, plain text).
+
+Going forward, we would like to announce an ongoing effort to improve these APIs to use something that is more modern, easy to discover learn and build API clients for.
```
Comment (arvindsv) [inline _posts/2015-06-10-Upcoming-API-Changes.markdown]: discover, learn ...

Comment (ketan) [inline _posts/2015-06-10-Upcoming-API-Changes.markdown]: fixed, rebased.

File: _posts/2015-06-10-Upcoming-API-Changes.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: Upcoming API Changes
+status: public
+type: post
+published: true
+author: Ketan Padegaonkar
+---
+
+With the upcoming release of Go 15.2, we'd like to begin unifying and improving some of the existing APIs that Go supports.
+
+Go's APIs are fairly old, have an [inconsistent and unpredictable content types](https://github.com/gocd/gocd/issues/572) (csv, xml, json, plain text).
```
Comment (arvindsv) [inline _posts/2015-06-10-Upcoming-API-Changes.markdown]: One more small change. Remove the "an" here.

Comment (ketan) [inline _posts/2015-06-10-Upcoming-API-Changes.markdown]: Resolved.

**Pull request merged**